site/irc.txt

```diff
@@ -0,0 +1,56 @@
+KISS IRC
+________________________________________________________________________________
+
+IRC was originally in the #kisslinux channel on Freenode, but was moved
+to Libera following the hostile behaviour on the part of Andrew Lee (see
+#/blog/20210512a).
+
+You can connect via Libera's webchat at https://web.libera.chat/#kisslinux.
+
+
+[1.0] Index
+________________________________________________________________________________
+
+- Public Logs                                                              [2.0]
+- IRC Etiquette                                                            [3.0]
+
+
+[2.0] Public Logs
+________________________________________________________________________________
+
+The IRC channel is publicly logged by a few of the channel denizens. Here are a
+list of public logs:
+
+- https://libera.irclog.whitequark.org/kisslinux/2021-05-29
```

Comment:
This probably doesn't need to point to a specific date?

Reply:
Whoops.